### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by pabliqe** [pfaffman/discourse-mailchimp-webhook#1]
Hello there! Is anyone using this plugin successfully? I’m having bad times trying to debug this since there’s no logs but Mailchimp is not adding users.

Any hint or workaround is really appreciated

**Comment by pfaffman** [comment]
The client that I developed this for is no longer using Discourse. Looking quickly at the code I don't see anything obvious that might have broken on the Discourse side of things. I won't have much time to support this if you don't have a budget.

Are you looking at `/var/discourse/shared/standalone/logs/rails/production.log`?

Have you entered the correct API key and list ID? The `mailchimp_dc` is a likely problem as I remember that it was non-obvious how to find it.

**Comment by pabliqe** [comment]
Yeah, I think the `mailchimp_dc` was ok, I got it from the secret key (last 2 chars) but wasn't clear if the whole key should be used on Discourse or this should be extracted.

I've tried both with no luck. No event or contacts triggered on Mailchimp on signups. I think the best workaround could be exporting users on CSV and uploading them on Mailchimp manually.

By the moment I'm no longer working on this instance so I'm going to close this issue.

Thanks @pfaffman!
